palo_alto_cortex_xdr/README.md

```diff
@@ -0,0 +1,42 @@
+# Agent Check: palo_alto_cortex_xdr
+
+## Overview
+
+This check monitors [Palo Alto Cortex XDR][1].
+
+## Setup
+
+### Installation
+
+The Palo Alto Cortex XDR check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. <List of steps to configure this integration>
```

Comment:
remove angle brackets from all aspects of the readme. It leads to issues when we try to sync it to create docs as it is treated as html

Reply:
Done.